### PR TITLE
Version 2

### PR DESCRIPTION
**Pull request by ericgaspar**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

**Comment by orhtej2** [comment]
I've pulled changes required for v0. 12 to #94, you're still lacking change in `upgrade`.

**Comment by tituspijean** [comment]
@ericgaspar If that's alright I'll rebase the branch on the testing one with v0.12.0

**Comment by tituspijean** [comment]
!testme

**Comment by tituspijean** [comment]
!testme

**Comment by tituspijean** [comment]
(to be restarted when https://github.com/YunoHost/yunohost/pull/1637 is released)

File: manifest.toml [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+sso = "not_relevant"
+disk = "50M"
+ram.build = "50M"
+ram.runtime = "50M"
+
+[install]
+    [install.warning]
+    ask.en = "Dendrite is still in beta phase.\nIt is functional, but all Matrix features are yet to be implemented.\nSee the upstream repo for more information: https://github.com/matrix-org/dendrite\n\nThis package est encore en development, all features from Matrix/Synapse are yet to be implemented."
+    ask.fr = "Dendrite est encore en phase bêta.\nElle est fonctionnelle, mais n'implémente pas encore toutes les fonctionnalités de Matrix.\nConsultez le dépôt source pour plus d'information : https://github.com/matrix-org/dendrite\n\nCe package est encore en développement, toutes les fonctionalités de Matrix/Synapse ne sont pas encore implémentées."
+    type = "display_text"
```
Comment (alexAubin) [inline manifest.toml:32]: (Maybe to be replaced with a `PRE_INSTALL.md`)

Comment (tituspijean) [reply manifest.toml]: Solved by eric ❤️

**Comment by tituspijean** [comment]
!testme

File: manifest.toml [hunk 2]
```diff
@@ -65,14 +65,14 @@ ram.runtime = "50M"
     [resources.permissions]
     main.url = "/"
     main.show_tile = false
-    server_api.url = "__DOMAIN__/_matrix "
```
Comment (tituspijean) [inline manifest.toml]: @ericgaspar this one was pretty tricky to find out, but these spaces were breaking the path detection so all requests were going through the SSO (and triggering CORS issues).

**Review by tituspijean — APPROVED** [review]

**Comment by ericgaspar** [comment]
@tituspijean why are we using

`https://api.github.com/repos/matrix-org/dendrite/tarball/v0.12.0`
over
`https://github.com/matrix-org/dendrite/archive/refs/tags/v0.12.0.tar.gz`

**Comment by tituspijean** [comment]
> @tituspijean why are we using
> 
> `https://api.github.com/repos/matrix-org/dendrite/tarball/v0.12.0`
> over
> `https://github.com/matrix-org/dendrite/archive/refs/tags/v0.12.0.tar.gz`

I think that was the link I used for the auto-updater v1. If the new one uses the `refs` link, I'm fine with that. :)

**Comment by ericgaspar** [comment]
ak ok. Let's see how the upgrade goes

**Pull request merged**